packages/big-design/src/components/Dropdown/spec.tsx

```diff
@@ -528,3 +528,27 @@ test('renders appropriate amount of list items', async () => {
 
   expect(listItems.length).toBe(3);
 });
+
+test('renders appropriate amount of line separators (HR)', async () => {
```

Comment:
We should also test for correct accessibility roles are enabled for this list item. Essentially test if it has a proper `aria-hidden`, `tabindex`, and can't focus on it.